cli/Cargo.toml

```diff
@@ -33,7 +33,7 @@ byteorder = "1.3.4"
 clap = "2.33.0"
 dirs = "2.0.2"
 dlopen = "0.1.8"
-dprint-plugin-typescript = "0.14.1"
+dprint-plugin-typescript = "0.16.0"
```

Comment:
Question: How do we tell people about `deno-fmt-ignore` and `deno-fmt-ignore-file`?

Reply:
There are two places we should do this: 

1. We should add a manual section for "deno fmt" in std/manual.md
2. I think adding it to the cli help messages would be helpful too https://github.com/denoland/deno/blob/796fc9bc3e8e4da7d53fb4617511ce4e2be22485/cli/flags.rs#L597-L604

Reply:
I updated the docs.

Note: there is a pending question here that should be considered https://github.com/denoland/deno/pull/5075#issuecomment-623594243